simphony_ui/couple_CFDDEM.py

```diff
@@ -25,38 +24,32 @@
 
 def run_calc(output_path, mesh_name):
     """Executes the full calculation"""
-    runstart = time.time()
-
-    mode_OF = "internal"
+    mode_cfd = "internal"
     mesh_type = "block"
 
     # Defining the wrapper for OpenFoam
-    if mode_OF == "internal":
-        wrapper_OF = openfoam_internal.Wrapper()
-        CUBAExt_OF = openfoam_internal.CUBAExt
-    elif mode_OF == "io":
-        wrapper_OF = openfoam_file_io.Wrapper()
-        CUBAExt_OF = openfoam_file_io.CUBAExt
+    if mode_cfd == "internal":
+        cfd_wrapper = openfoam_internal.Wrapper()
+        cuba_ext_cfd = openfoam_internal.CUBAExt
+    elif mode_cfd == "io":
+        cfd_wrapper = openfoam_file_io.Wrapper()
+        cuba_ext_cfd = openfoam_file_io.CUBAExt
     else:
-        print "Wrong mode_OF!"
+        print "Wrong mode_cfd!"
         sys.exit(1)
 
     # define the wrapper for LIGGGHTS
     dem_wrapper = liggghts.LiggghtsWrapper(use_internal_interface=True)
 
-    if mode_OF == "internal":
-        path = output_path
-        print path
-
     # General settings
     number_iterations = 10
 
     force_type = "Stokes"
 
     # OF settings
-    num_timesteps_OF = 10
-    timestep_OF = 2.0e-4
-    visco_OF = 1.0e-3
+    num_timesteps_cfd = 10
```

Comment:
openfoam again. We want to be clear that the application uses openfoam and ligghts. However, OF as an acronym is not very meaningful and immediate, so expanding it is a good idea.